src/components/Loader/Loader.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { Meta } from '@storybook/react';
+import Loader from './Loader';
+
+const meta = {
+  title: 'Components/Loader',
+  component: Loader,
+  argTypes: {
+    isCentered: {
```

Comment:
I'm not seeing a difference when toggling `isCentered` between true and false. Is there another story you can add that shows when you'd want to set `isCentered` to false?